by-language/java-jooq/backlog.rst

```diff
@@ -0,0 +1,69 @@
+###################################################
+Backlog for Java jOOQ demo application with CrateDB
+###################################################
+
+
+Features
+========
+
+- Evaluate more features of jOOQ, like:
+
+  - `Using JDBC batch operations`_
+  - `jOOQ for PROs`_, specifically but not exclusively `exporting`_ and `importing`_
+  - `Alternative execution models`_
+  - `The static query DSL API`_
+  - `Configuration and setup of the generator`_
+
+- jOOQ's code generator does not know about CrateDB's special data types.
+  Evaluate if this will work when using the dynamic access variant, without
+  using code generation. Of course, the container data types ``OBJECT`` and
+  ``ARRAY`` are the most interesting ones.
+
+- How to set the default schema name? The ``unqualifiedSchema`` property in
+  ``jooq.gradle`` apparently only accepts ``public`` or ``none``.
```

Comment:
Coming from https://github.com/crate/cratedb-examples/pull/11#discussion_r1086385467, I've added this backlog item.

Reply:
When thinking about it another time, it is probably only applicable when connecting to a real database for the code generation step (GH-10). In this scenario, the corresponding setting is probably `jooq.configurations.main.generationTool.generator.database.inputSchema`.

https://github.com/crate/cratedb-examples/blob/f88eda5622688254a9f86d29575952393fc7b3bd/by-language/java-jooq/jooq.gradle#L70-L73

Reply:
Now also documented at [CrateDB/jOOQ backlog » Q & A](https://github.com/crate/cratedb-examples/blob/amo/jooq/improve-docs/by-language/java-jooq/backlog.rst#q--a).